spring-grpc-docs/pom.xml

```diff
@@ -20,17 +20,14 @@
 	<!-- Dependencies used to build the config props doc generator -->
 	<dependencies>
 		<dependency>
-			<groupId>org.springframework</groupId>
-			<artifactId>spring-core</artifactId>
+			<groupId>org.springframework.grpc</groupId>
+			<artifactId>spring-grpc-spring-boot-autoconfigure</artifactId>
+			<version>${project.version}</version>
 		</dependency>
 		<dependency>
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-databind</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.springframework.grpc</groupId>
-			<artifactId>spring-grpc-spring-boot-autoconfigure</artifactId>
-			<version>${project.version}</version>
+			<version>${jackson.version}</version>
```

Comment:
This used to come in from ROOT/pom.xml depMgmt on spring-boot-dependencies but needs to come out as Spring Boot will not exist in this repo eventually (other than ITs and samples).